argo-cd-apps/base/host/segment-bridge/segment-bridge-host.yaml

```diff
@@ -17,8 +17,8 @@ spec:
               elements:
                 - nameNormalized: stone-stg-host
                   values.clusterDir: stone-stg-host
-                - nameNormalized: stone-prd-host
-                  values.clusterDir: stone-stg-host
+                - nameNormalized: stone-prd-host1
+                  values.clusterDir: stone-prd-host1
```

Comment:
you also need to rename the folder `components/segment-bridge/production/stone-prd-host` to `components/segment-bridge/production/stone-prd-host1`